pac4j-oidc/src/test/java/org/pac4j/oidc/credentials/authenticator/OidcAuthenticatorTest.java

```diff
@@ -0,0 +1,80 @@
+package org.pac4j.oidc.credentials.authenticator;
+
+import static org.junit.Assert.assertEquals;
+
+import com.nimbusds.oauth2.sdk.auth.ClientAuthenticationMethod;
+import com.nimbusds.oauth2.sdk.id.Issuer;
+import com.nimbusds.openid.connect.sdk.SubjectType;
+import com.nimbusds.openid.connect.sdk.op.OIDCProviderMetadata;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Test;
+import org.pac4j.core.exception.TechnicalException;
+import org.pac4j.oidc.client.OidcClient;
+import org.pac4j.oidc.config.OidcConfiguration;
+
+public class OidcAuthenticatorTest {
```

Comment:
Can you add some javadoc to the class header (`@author` and `@since` at least)?

Reply:
Yes sure, fixed in cfee7886c869ca4cb61c9cf820302ecbd6b3a9e2 :)

Reply:
Good. Please submit the same PR for the master branch.

Reply:
Opened #2510 for porting the changes to the master branch

Reply:
Excellent! Thanks